pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -159,8 +159,9 @@ func (b *ConsoleServerCLIConfigBuilder) AuthConfig(authnConfig *configv1.Authent
 		}
 
 		oidcProvider := authnConfig.Spec.OIDCProviders[0]
-		oidcConfig := util.GetOIDCClientConfig(authnConfig)
+		oidcConfig, _ := util.GetOIDCClientConfig(authnConfig)
```

Comment:
this does not appear to belong to this commit

Reply:
I've moved the `b.authType = "disabled"` bit to its relevant commit (408923f24851f7606d6cd7076c5f1d2d423bb26f).

The signature of `GetOIDCClientConfig()` is changed though in commit d3813df4405a9ec94ab937f43f269bfeb0fee734 -- would it be better to split the signature change in a separate commit?